cmd/pipeline/deploy.go

```diff
@@ -186,8 +185,14 @@ func getPipelineName() (string, error) {
 	if err != nil {
 		return "", err
 	}
-
-	return filepath.Base(workDir), nil
+	name, err := model.GetValidNameFromGitRepo(workDir)
```

Comment:
I think we need to do the same for destroy

Reply:
This method is also called from destroy